includes/shortcodes.php

```diff
@@ -6,8 +6,96 @@
 add_shortcode('tsml_meeting_count', 'tsml_count_meetings');
 add_shortcode('tsml_region_count', 'tsml_count_regions');
 
+//output the Legacy UI meeting finder
+//example:  [tsml_query_string key="type" value="women"]
+function tsml_legacy_ui($atts, $content = null)
```

Comment:
i would prefer not to add a shortcode for "legacy ui" since it wasn't requested, and we are trying to move away from legacy ui. could we remove this option?

legacy ui doesn't really lend itself to being a shortcode because it's too complex. - it won't be the full thing with filters and search, etc, and clicking back to meetings will take you to a different page

Reply:
Yep. I expected you wouldn't like it but it was just so easy to do. I'll get rid of it.

Reply:
Just for the record, I don't support moving away from TSML. I think the majority of sites are based on it and we have a long ways to go before TSML UI has all the features which would allow TSML to be depricated...